taichi/gui/win32.cpp

```diff
@@ -53,7 +53,7 @@ static std::string lookup_keysym(WPARAM wParam, LPARAM lParam) {
       if (VK_F1 <= key && key <= VK_F12)
         return fmt::format("F{}", key - VK_F1);
       else if (isascii(key))
-        return std::string(1, key);
+        return std::string(1, ::tolower(key));
```

Comment:
`::tolower` seems to be the C legacy function. Could you `#include <cctype>` and use `std::tolower` instead?

Reply:
FYI the `isascii` above is also a C legacy function..
Also note that I found the *modern-featured-fasion-advanced-highly-superior* `<cctype>` says:
```
namespace std
{
  using ::isalnum;
  using ::isalpha;
  using ::iscntrl;
  using ::isdigit;
  using ::isgraph;
  using ::islower;
  using ::isprint;
  using ::ispunct;
  using ::isspace;
  using ::isupper;
  using ::isxdigit;
  using ::tolower;
  using ::toupper;
} // namespace std
```
cctype = copy from ctype :rofl: